ooniprobe/AppDelegate.m

```diff
@@ -28,6 +29,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [[UIBarButtonItem appearance] setTitleTextAttributes:[NSDictionary dictionaryWithObjectsAndKeys: [UIFont fontWithName:@"FiraSans-Regular" size:16],NSFontAttributeName, nil] forState:UIControlStateNormal];
     [NavigationBarUtility setDefaults];
     [CountlyUtility initCountly];
+    [ReachabilityManager sharedManager];
```

Comment:
Is the purpose of this statement to create a singleton instance? If so, could you please document it?

Reply:
Yes, to create the singleton. Just the comment will be enough?

Reply:
Absolutely, thank you!